### PR TITLE
[MODULES-4598] Revert "Revert "fix default params for SUSE family OSes""

### PR DESCRIPTION
**Pull request by mmoll**
This reverts commit 63ad3437f291c378c6ae95ea30ac4be5de0c7734, which reverted
commit b6fbbb3600f8bab904cdb980aa429b56b731443e.

### CODE REVIEW

**Comment by mmoll** [comment]
@wilson208 @eputnam 
this is working fine for me on SLES 12 SP2

**Comment by eputnam** [comment]
@mmoll I'm going to run this through our CI again and try to see what was up.

File: manifests/globals.pp [hunk 1]
```diff
@@ -111,10 +111,10 @@
     'Suse' => $::operatingsystem ? {
       'SLES' => $::operatingsystemrelease ? {
         /11\.[0-4]/ => '91',
-        default => '93',
+        default => '94',
```
Comment (eputnam) [inline manifests/globals.pp]: @mmoll turns out we need a separate condition. SLES 12.0 needs to stay at 93 but SLES =>12.2 works with 94.

Comment (mmoll) [inline manifests/globals.pp]: I see.. I'll whip something up

**Comment by mmoll** [comment]
I checked on the 12SP1 DVD that SP1 already has postgresql94, the second commit should fix that

**Comment by eputnam** [comment]
thanks @mmoll !

**Pull request merged**